lib/DDG/Spice/Airlines.pm

```diff
@@ -8,8 +8,7 @@ spice to => 'https://api.flightstats.com/flex/flightstatus/rest/v2/jsonp/flight/
 spice from => '(.*)/(.*)/(.*)/(.*)/(.*)/(.*)';
 spice proxy_cache_valid => '418 1d';
 
-triggers any => '///***never trigger***///';
-# triggers query_lc => qr/^(\d+)\s+(.*?)(?:[ ]air.*?)?$|^(.*?)(?:[ ]air.*?)?\s+(\d+)$/;
```

Comment:
This should remain commented out -- we have a special internal trigger for this Instant Answer